Gemfile

```diff
@@ -54,6 +54,3 @@ group :development do
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
-
-# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
```

Comment:
I was getting errors in my bundler and it was because I need to upgrade my ruby version. After upgrading the version this solved itself.